docs/user/communication.rst

```diff
@@ -5,21 +5,25 @@ Communication
 
 .. contents:: Content of this document
     :local:
-    :depth: 2
+    :depth: 3
 
 Artemis enables students, tutors, and instructors to actively participate with its communication feature.
-Various communication features allow students to engage with peers and ask all kinds of questions whereas moderators (e.g., instructors and tutors) can provide general course information and answer content-related questions.
-Communication can be made in different contexts, namely for lectures, exercises, or courses.
-Below, you can find more information on specific features and how to use them.
+Various communication features allow students to engage with peers and ask all kinds of questions whereas moderators
```

Comment:
The sentence before speaks from one feature, now there are multiple ones

Reply:
This was done by previous developers and reformatted by me. However, I updated the first sentence to fit in better with the rest of the paragraph. How does it look now?